doc/api/globals.md

```diff
@@ -213,7 +213,7 @@ added: v10.0.0
 
 The WHATWG `URLSearchParams` class. See the [`URLSearchParams`][] section.
 
-## WebAssembly
+## `WebAssembly`
```

Comment:
a single-word title with code background looks weird

Reply:
The CSS will prevent code background in the HTML docs.